config/config.go

```diff
@@ -107,5 +109,13 @@ func ConfigurationFromEnvironment() *Configuration {
 		ret.SendFunctionLogs = true
 	}
 
+	if sendFunctionLogsOverride && sendFunctionLogsStr == "true" {
+		ret.SendFunctionLogs = true
+	}
+
+	if logsEnabledOverride && logsEnabledStr == "false" {
+		ret.LogsEnabled = false
```

Comment:
The empty value for booleans  is `false`. Looks like logs would be disabled here without  a way to enable them?

Reply:
@kolanos, let me know if I'm misunderstanding, Go isn't my strongest language.  

`ret.LogsEnabled` is initialized to `true` on line 52.  The idea here was to maintain current behavior (logging enabled) while allowing users to explicitly opt out of logging by setting `NEW_RELIC_EXTENSION_LOGS_ENABLED` to `"false"`.  This block would only execute in that scenario.

Reply:
@codylandry Ah, Didn't see it on line 52. Can you refactor so that the block is consistent with the `extensionEnabled` block? (move it up before struct creation, have a variable for this state, etc.

Reply:
No problem, just pushed the change.  I also added the `strings.ToLower` call as well.  Thanks, I'd missed the similarity there.